shortstories/templates/shortstories/shortstories_edit.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'base/index.html' %}
+
+{% block content %}
+    <h1>New Short Story</h1>
+    {% if form.errors %}
+        <p style="color: red;">
+            Please correct the error{{ form.errors|pluralize }} below.
+        </p>
+        {% endif %}
+    <form method="POST" class="post-form" novalidate>{% csrf_token %}
```

Comment:
It's better to include form action here, please check this https://www.w3schools.com/Tags/att_form_action.asp

Reply:
Also why did you use novalidate?